omo_r1mini_mcu_node:
  ros__parameters:
    port:
      name: "/dev/ttyMCU"
      baudrate: 115200

    wheel:
      separation: 0.17
      radius: 0.0335

    motor:
      gear_ratio: 21.3
      max_lin_vel_x: 2.0
      max_rot_vel_z: 2.5
      
    sensor:
      enc_pulse: 1440.0

diff_drive_controller:
  ros__parameters:
    odometry:
      frame_id: "odom"
      child_frame_id: "base_footprint"
